rs/interfaces/src/crypto/keygen.rs

```diff
@@ -23,7 +23,7 @@ pub trait KeyManager {
     /// * [`AllKeysRegistered`]: Registry contains all required public keys and
     ///   secret key store contains all corresponding secret keys.
     ///
-    /// * [`AllKeysRegistered`]: PublicKeyRegistrationStatus::AllKeysRegistered
+    /// * [`AllKeysRegistered`][]: PublicKeyRegistrationStatus::AllKeysRegistered
```

Comment:
Not sure what the purpose of the additional `[]` is, but this entire paragraph seems outdated anyway, so if possible, just remove it. In particular, the following can be removed:
```
    /// Returns the status of the public keys as follows:
    /// * [`AllKeysRegistered`]: Registry contains all required public keys and
    ///   secret key store contains all corresponding secret keys.
    ///
    /// * [`AllKeysRegistered`][]: PublicKeyRegistrationStatus::AllKeysRegistered
```

Reply:
done